packages/addons/service/nut/patches/nut-lowspeed-buffer-size.patch

```diff
@@ -0,0 +1,19 @@
+http://bugs.gentoo.org/show_bug.cgi?id=318937
```

Comment:
patches should be numbered to guarantee application sequence, e.g. `0001-nut-lowspeed-buffer-size.patch`

Reply:
Fixed